Octokit.Reactive/Clients/ObservablePullRequestReviewRequestsClient.cs

```diff
@@ -0,0 +1,47 @@
+using System;
+using System.Reactive;
+using System.Reactive.Threading.Tasks;
+using Octokit.Reactive.Internal;
+
+namespace Octokit.Reactive
+{
+    public class ObservablePullRequestReviewRequestsClient : IObservablePullRequestReviewRequestsClient
+    {
+        readonly IPullRequestReviewRequestsClient _client;
+        readonly IConnection _connection;
+
+        public ObservablePullRequestReviewRequestsClient(IGitHubClient client)
+        {
+            Ensure.ArgumentNotNull(client, "client");
+
+            _client = client.PullRequest.ReviewRequests;
+            _connection = client.Connection;
+        }
+
```

Comment:
Just FYI that we add the xmldoc comments to the interfaces and the client implementations, so once you are done with the changes please add the xmldoc comments to all the conrete methods as well :+1: